database/src/main/flow_patterns/checkpoint_measure_definitions.ddl

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- DROP TABLE IF EXISTS flow_patterns.checkpoint_measure_definitions;
+
+CREATE TABLE flow_patterns.checkpoint_measure_definitions
+(
+    id_fp_checkpoint_measure_definition BIGINT NOT NULL DEFAULT global_id(),
+    key_fp_flow                         BIGINT NOT NULL,
+    measure_type                        TEXT NOT NULL,
+    measure_fields                      TEXT[] NOT NULL,
+    created_by                          TEXT NOT NULL,
+    created_when                        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    updated_by                          TEXT NOT NULL,
+    updated_when                        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
```

Comment:
```suggestion
    created_at                        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
    updated_by                          TEXT NOT NULL,
    updated_at                        TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
```
Just suggesting a rename that might be more fitting, this does not change the logic, of course.

This could be also reflected elsewhere, there are many more fields like this.